<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelect</class>
 <widget class="QWidget" name="ColorSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="r">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxR">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="g">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxG">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="b">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxB">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="displayWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Да</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>g</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>g</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>b</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>r</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>r</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
